doc/apiary/v2/fiware-ngsi-simple-cookbook.apib

```diff
@@ -0,0 +1,872 @@
+FORMAT: 1A
```

Comment:
no problem, for consistency I can rename but I will do it before landing this PR